source/_components/climate.sensibo.markdown

```diff
@@ -17,12 +17,14 @@ Integrates [Sensibo](https://sensibo.com) Air Conditioning controller into Home
 
 To enable this platform, add the following lines to your `configuration.yaml` file:
 
+{% configuration %}
 ```yaml
 # Example configuration.yaml entry
 climate:
   - platform: sensibo
     api_key: <your_key_here>
 ```
+{% endconfiguration %}
```

Comment:
`{% configuration %}` is for the [variables](https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration) and not the sample itself.

Reply:
On it, sorry for the misunderstanding